src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,10 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migrating from `io.sentry:sentry-android 6.2.x` to `io.sentry:sentry-android 6.3.0`
+
+* `sentry-android-okhttp`, `sentry-android-fragment` and `sentry-android-timber` packages now declare their upstream dependencies as `compileOnly`. Please make sure to have `okhttp`, `fragment` and `timber` dependencies on the classpath if you are using those integrations.
```

Comment:
```suggestion
* `sentry-android-okhttp`, `sentry-android-fragment` and `sentry-android-timber` packages now declare their upstream dependencies as `compileOnly`. Please make sure to have `okhttp`, `fragment` and `timber` dependencies on the classpath if you are using those integrations, otherwise, the application will crash with a `NoClassDefFoundError` exception.
```